clients/python/README.md

```diff
@@ -3,7 +3,9 @@
 - Python 3
 
 ## Setup
-`pip install -r requirements.txt`
+1. `pip install -r requirements.txt`
```

Comment:
I'm pretty sure it's global by default so it doesn't matter here. You could pass `--user`.